anyway/infographics_utils.py

```diff
@@ -1233,6 +1233,8 @@ def localize_items(request_params: RequestParams, items: Dict) -> Dict:
         }
         return items
 
+# adding calls to _() for pybabel extraction
+_("Injured and killed pedestrians by severity and year")
```

Comment:
Where is this text used?

Reply:
in the self.information = "Injured and killed pedestrians by severity and year"

Reply:
Sorry, I missed it.